src/utils/pause/frames/displayName.js

```diff
@@ -82,13 +82,22 @@ export function formatDisplayName(
   frame: LocalFrame,
   { shouldMapDisplayName = true, maxLength = 25 }: formatDisplayNameParams = {}
 ) {
-  let { displayName, originalDisplayName, library } = frame;
-  displayName = originalDisplayName || displayName;
+  let {
+    displayName,
+    originalDisplayName,
+    userDisplayName,
+    name,
+    library
+  } = frame;
+  displayName = originalDisplayName || userDisplayName || displayName || name;
```

Comment:
this seems like an opportunity to consolidate some logic the `utils/frame` dir

```js
function getDisplayName(frame: Frame) {
  let {
    displayName,
    originalDisplayName,
    userDisplayName,
    name,
    library
  } = frame;
  return originalDisplayName || userDisplayName || displayName || name;
}
```

Reply:
sounds good indeed